usaspending_api/references/tests/data/test_load_ref_country_pop_data.csv

```diff
@@ -0,0 +1,210 @@
+country_code,population
```

Comment:
Unit tests look good. I don't see where this file is being used though. Maybe originally intended for an integration test?

Reply:
removed the unused file.